httpclient/src/test/kotlin/tbdex/sdk/httpclient/TbdexHttpClientTest.kt

```diff
@@ -84,7 +87,20 @@ class TbdexHttpClientTest {
   }
 
   @Test
-  fun `send RFQ success via mockwebserver`() {
+  fun `send RFQ without replyTo success via mockwebserver`() {
+
+    server.enqueue(MockResponse().setResponseCode(HttpURLConnection.HTTP_ACCEPTED))
+
+    val rfq = TestData.getRfq(pfiDid.uri, TypeId.generate("offering"))
+    assertDoesNotThrow { TbdexHttpClient.sendMessage(rfq) }
+
+    val request1 = server.takeRequest()
+    assertEquals(request1.path, "/exchanges/${rfq.metadata.exchangeId}")
+    assertEquals(request1.body.readUtf8(), Json.stringify(mapOf("rfq" to rfq)))
```

Comment:
What do we think of testing path and url this way? Other endpoints will also have http headers that we need to test for, but this one does not.

If we like this pattern, I will open a follow up PR which adds these assertions to all relevant http client tests.

Reply:
The assertion of the path is fine, but the assertion of the body may lead to weird false negatives because I'm not sure we can safely assume the `.readUtf8()` and `Json.stringify()` functions will encode the string in the exact same ways (you could imagine one using spaces, and the other tabs, or any weird whitespace thing, or non-whitespace special characters such as `\n`)

But, we may be doing this all over the place, in which case I wouldn't worry about it, at least right now, and also I'm not a Kotlin expert so this skepticism may be totally misplaced

Reply:
Here's an alternative that addresses potential differences in whitespace. We can JSON parse the actual and expected request bodies to get `JsonNode` objects which we can deep compare.

```kotlin
assertEquals(
  Json.jsonMapper.readTree(request1.body.readUtf8()),
  Json.jsonMapper.readTree(Json.stringify(mapOf("rfq" to rfq)))
)
```

Reply:
Hmmmm better but still don't _love_ it

Does the `assertEquals()` do a deep compare on any Kotlin object? If so, you could parse the http body into an RFQ object and just do a assertion with the existing `rfq` variable. This would also technically test our parsing functionality which is a little against the concept of a unit test.

Yeah I'm cool with any of the above approaches... how it is now is fine, your proposal is fine, or what I just ideated on with parsing. Your call!